pkg/ensurance/executor/throttle.go

```diff
@@ -140,7 +141,7 @@ func (t *ThrottleExecutor) Avoid(ctx *ExecuteContext) error {
 			}
 
 			if limitCPU, ok := container.Resources.Limits[v1.ResourceCPU]; ok {
-				if float64(limitCPU.MilliValue())/1000.0*float64(throttlePod.CPUThrottle.MinCPURatio) > containerCPUQuotaNew {
+				if float64(limitCPU.MilliValue())/1000.0*float64(throttlePod.CPUThrottle.MinCPURatio)/100 > containerCPUQuotaNew {
```

Comment:
can 1000.0 and 100 to be defined as const?

Reply:
Done